source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/__tests__/scenarios.ts

```diff
@@ -0,0 +1,93 @@
+export const threeProjects = {
+  data: [{
+    type: 'projects',
+    id: '1',
+    attributes: {
+      'name': 'Dummy project',
+      'date-archived': null,
+      'language': 'English',
```

Comment:
Do we need this extra attributes in order to test what you need?

Reply:
since this is a scenario that could be used for other tests, (specifically filtering by language (coming up next!), it should stay :)